pages/index.tsx

```diff
@@ -39,6 +46,27 @@ export default function Home () {
           loading={loading}
           getData={getData}
         />
+        <Input
+          value={valueThree}
+          setValue={setValueThree}
+          loading={loading}
+          getData={getData}
+        />
+        <Input
+          value={valueFour}
+          setValue={setValueFour}
+          loading={loading}
+          getData={getData}
+        />
+</div>
+
+<div className={styles.playerCard}>
+  <div className={styles.playerCardValue}><button>-</button><p>Life Total:</p>40<p></p><button>+</button></div>
```

Comment:
All of these could be components i think 🤔

Reply:
I was thinking that may be the case, I'll work on that next